### ISSUE TITLE
"Analyze" function ceased to function

### ISSUE THREAD

**Issue by CaMer0n** [TB-Electronics/fscan#5]
It was working for a while, but now it does not scan/process the DIRP results.

**Comment by goldengel** [comment]
It depends on the frequencies in Editor. Please check: Frequencies + selected database. Now analyze always works.

**Comment by CaMer0n** [comment]
![img_1039](https://cloud.githubusercontent.com/assets/2825679/25766312/acb5f1fe-31a6-11e7-8847-67ab81b960ae.JPG)

![img_1040](https://cloud.githubusercontent.com/assets/2825679/25766321/b2758ffa-31a6-11e7-941a-883731861f17.JPG)

![img_1041](https://cloud.githubusercontent.com/assets/2825679/25766325/b8266ae6-31a6-11e7-9358-709df2ab4a68.JPG)

btw. Clicking "Start" does not do anything.

**Comment by goldengel** [comment]
1. Do the Dirp.
2. After Dirp, click "Transfer" Slider (all frequencies will be transferred to editor.
3. Click Analyze button (it will compare these frequencies with database)

**Comment by CaMer0n** [comment]
Thank you! So is the "PLAY" button ">" on the left meant to be a combination of these 3 steps? Or only the 3rd step?

**Comment by goldengel** [comment]
You are right, this is not described well in the manual. The templates / scripts are to automate stuff. you can create a simple script to just setup a DIRP range. Or huge scripts to fully start an automated sequence. You can:
1. Click the script button > Click the Play button to start this script
2. Click the script button > Click the same script button again to start this script


![screenshot - 2017_05_12 03-05-28](https://cloud.githubusercontent.com/assets/1375893/25999200/750370dc-3724-11e7-97c5-a2a64a4bdeb9.jpg)

**Comment by CaMer0n** [comment]
Okay, thank you!
Intuitively it felt more like it should run the DIRP and Analysis of the tabs to the right. eg. "Clark"
